sdk/textanalytics/azure-ai-textanalytics/samples/sample_analyze_healthcare_entities.py

```diff
@@ -74,41 +62,24 @@ def analyze_healthcare_entities(self):
 
         docs = [doc for doc in result if not doc.is_error]
 
-        print(
-            "In order to find the total dosage for every mentioned medication, "
-            "let's create a dict, mapping medication name -> total dosage. "
-        )
-
-        medication_to_dosage = defaultdict(int)
-
-        print(
-            "We will start off by extracting all of the dosage entities."
-        )
-
-        dosage_entities = [
-            entity
-            for doc in docs
-            for entity in doc.entities
-            if entity.category == "Dosage"
-        ]
-
-        print(
-            "Now we traverse the related entities of each dosage entity. "
-            "We are looking for entities that are related by 'DosageOfMedication'. "
-            "After that, we're done!"
-        )
-        for dosage in dosage_entities:
-            dosage_value = int(re.findall(r"\d+", dosage.text)[0]) # we find the numbers in the dosage
-            for related_entity, relation_type in dosage.related_entities.items():
-                if relation_type == "DosageOfMedication":
-                    medication_to_dosage[related_entity.text] += dosage_value
-
-        [
-            print("We have fulfilled '{}' total mg of '{}'".format(
-                dosage, medication
-            ))
-            for medication, dosage in medication_to_dosage.items()
-        ]
+        print("Results of Healthcare Entities Analysis:")
```

Comment:
I liked it more when the sample had a use case instead of printing data... but I will leave that to Python to decide

Reply:
yeah, I can add back some of that. The reason why I scaled back was bc for the ux study we used it for, the person thought the sample was the only way you could interact with the service basically. Let me see how much I can add back

Reply:
oh nooo... ok. leave as u have it, and after ux studies we can have a discussion about our samples story